README.md

```diff
@@ -44,15 +44,18 @@ Unfortunately, googletest and lapack are not as easy to install.  Many distros d
 #### (optional, one time only)
 1.  `mkdir -p [ROCBLAS_BUILD_DIR]/release/deps`
 2.  `cd [ROCBLAS_BUILD_DIR]/release/deps`
-3.  `cmake [ROCBLAS_SOURCE]/deps`
-4.  `make -j$(nproc) install`
+3.  `cmake -DBUILD_BOOST=OFF [ROCBLAS_SOURCE]/deps`
```

Comment:
I know that our jenkinsfile doesn't build boost because it installs boost from apt, but I don't want to give advice to users NOT to build boost.  I default it on because it should be safe to build boost no matter if they need it or not.  The only penalty is that they are using extra hard drive space.